### DISCUSSION TITLE
Translations

### DISCUSSION THREAD

**Discussion by hammerdirt-analyst** [hammerdirt-analyst/IQAASL-End-0f-Sampling-2021#11] [General]
### Lets not lose things in translation

How these reports are translated is central to how they are perceived. This is the place for the suggestions and comments.

**Comment by hammerdirt-fieldops** [comment]
Changes column names in german reports:
1. Summary data and material types in all survey area charts: change Ubekannt to **Unbekannt**
2. Change Quatre Cantons to **Vierwaldstättersee** in tables and section 4.1 seen
3. Vorwort: Change fig 1 heat map afeaturename to thunersee-breienzersee
4. Survey results and land use column headers: change % zu Gebäude to **% zu Gebäuden**. Change kreuzt to **Intersekte**

Define intersect in both english & german versions
English = *Intersects are discharge points from rivers, streams, and canals within 1500 m of the survey site. 
German = *Intersekte sind Einleitungsstellen von Flüssen, Bächen und Kanälen im Umkreis von 1500 m um den Erhebungsort. 

Add to acronyms LWS = Landwirtschaft

Linth survey results and land use: Reword English and German: There are less than 100 samples, proceed with caution. Beach litter surveys have alot of variance change to **The Linth, Limmat survey area has less than 100 samples and considerable variance.**
German: **Das Untersuchungsgebiet Linth, Limmat umfasst weniger als 100 Proben und weist eine erhebliche Varianz auf.**

**Reply by hammerdirt-analyst** [reply]
The following were addressed and fixed in the next build. Issues 12, 13, 14 and 15. The definition of the an intersection is given with the land use profiles.

The less than 100 sample warning is just a warning to let people know that the do not have alot of samples.

**Comment by hammerdirt-fieldops** [comment]
Edits:
German version Aare survey area,  map of survey area change Müllstücken to **Abfallobjekten**
English version Aare survey area,  map of survey area add **Marker diameter = the mean survey result in pieces of litter per 100 meters (p/100m).**

**Reply by hammerdirt-analyst** [reply]
These changes have been added issues 16 and 17. The english version of the templates for survey area and water feature was also corrected.

**Comment by hammerdirt-analyst** [comment]
More correction in the Survey area template:

intersects: Flussmündung
Rose: Rosa
Strandabfälle:Strand-Abfallobjeket
Right: Rechts

**Reply by hammerdirt-analyst** [reply]
Issue 18 see next build for corrections. 

**Changing intersects to Flüssmundung will require changes in every other chapter and several methods.**

**Comment by hammerdirt-analyst** [comment]
Changing the name of chapter 21 needs to be discussed. The translation is not the same. In English we are saying there was more and less when both sampling periods are considered, depending on which object you are talking about.

The following changes were made:

remove stray english phrase
add missing translation
rose:rosa
Interpretation Sie Spearmans: Interpretation von Spearmans
correct inverted phrase

**Comment by hammerdirt-analyst** [comment]
Changes to chapter 15, 16 and 21:

Wahrscheinlich : Wahrscheinlichkeit

Microplastics : Mikrokunststoffe

Infrastructure : Infrastruktur

insert missing translation.

reference: [main 2168c42]
